### PR TITLE
[Fix](executor)Fix routine load failed when can not find group

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
```
ErrorReason{code=errCode = 2, msg='failed to create task: errCode = 2, detailMessage = can not find workload group null'}
```

Fix routineload may failed because of can not find workload group.

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java [hunk 1]
```diff
@@ -223,16 +223,22 @@ public List<TPipelineWorkloadGroup> getTWorkloadGroupById(long wgId) {
...
         String groupName = Env.getCurrentEnv().getAuth().getWorkloadGroup(user.getQualifiedUser());
         List<TPipelineWorkloadGroup> ret = new ArrayList<>();
         readLock();
         try {
             WorkloadGroup wg = nameToWorkloadGroup.get(groupName);
             if (wg == null) {
-                throw new UserException("can not find workload group " + groupName);
+                LOG.warn("can not find workload group " + groupName + " for user " + user.getQualifiedUser());
+                wg = nameToWorkloadGroup.get(DEFAULT_GROUP_NAME);
+                if (wg == null) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java:235]: How can this happen？

**Comment by wangbo** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**